src/index.html

```diff
@@ -25,26 +27,49 @@
         class="text-center p-2 bg-green-300 hover:bg-green-400 transition rounded-r"><i style="color: black;"
           class="fa fa-times-circle"></i></button>
     </div>
-
   </div>
-  <div class="w-full min-h-3 p-2 flex justify-center pt-4">
-    <button id="back" onClick="backward()" class="text-center p-2 bg-blue-300 hover:bg-blue-400 rounded-l transition"><i
+  <!-- Utility buttons -->
+  <div class="w-full min-h-3 p-2 flex justify-center pt-4 dark:text-gray-900">
+    <button id="back" onclick="backward()" class="text-center p-2 bg-blue-300 hover:bg-blue-400 rounded-l transition"><i
         style="color: black;" class="fa fa-backward"></i></button>
-    <button id="forward" onClick="forward()" class="text-center p-2 bg-green-300 hover:bg-green-400 transition"><i
+    <button id="forward" onclick="forward()" class="text-center p-2 bg-green-300 hover:bg-green-400 transition"><i
         style="color: black;" class="fa fa-forward"></i></button>
-    <input id="searchbar" type="text" style="width: 90%" class="min-w-fit bg-gray-100 p-2"
+    <input id="searchbar" type="text" style="width: 90%" class="min-w-fit bg-gray-100 dark:text-gray-900 p-2"
       placeholder="Browse the Web" />
-    <button id="go" onClick="loadURL()" class="text-center p-2 bg-purple-300 hover:bg-purple-400 transition">Go</button>
-    <button id="reload" onClick="reload()" class="text-center p-2 bg-red-300 hover:bg-red-400 transition rounded-r"><i
+    <button id="go" onclick="loadURL()" class="text-center p-2 bg-purple-300 hover:bg-purple-400 transition">Go</button>
+    <button id="reload" onclick="reload()" class="text-center p-2 bg-red-300 hover:bg-red-400 transition"><i
```

Comment:
_Apparently_ my VSCode seems to like `onclick` more than `onClick` so I changed it, as I think in the HTML spec it is supposed to be `onclick`.